README.md

```diff
@@ -1061,6 +1061,10 @@ VRF member of the interface.  Valid values are a string or the keyword 'default'
 
 ##### SVI interface config attributes
 
+###### `fabric_forwarding_anycast_gateway`
+Associate SVI with anycast gateway under VLAN configuration mode.
+Valid values are 'true', 'false', and 'default'.
```

Comment:
Does it makes sense to add a note here that you must set `cisco_overlay_global` `anycast_gateway_mac` before setting this property?